src/app/pages/checkout-review/checkout-review/checkout-review.component.ts

```diff
@@ -16,16 +17,17 @@ export class CheckoutReviewComponent implements OnInit {
   @Input() submitting: boolean;
   @Output() createOrder = new EventEmitter<void>();
 
-  form: FormGroup;
+  form = new FormGroup({});
+  model: {};
```

Comment:
use the model to initialize the field with false, otherwise you can skip the model at all